compute/bare-metal-cloud/how-to/order-flexible-ip.mdx

```diff
@@ -11,7 +11,7 @@ dates:
   posted: 2021-05-26
 ---
 
-[Flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
+[flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
```

Comment:
Is this the start of a sentence though, should it still be capitalised?

Reply:
```suggestion
[Flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
```